diff --git a/evm/src/NttManager/NttManager.sol b/evm/src/NttManager/NttManager.sol
index 7ac021b..60e4aee 100644
--- a/evm/src/NttManager/NttManager.sol
+++ b/evm/src/NttManager/NttManager.sol
@@ -383,7 +383,7 @@ contract NttManager is INttManager, RateLimiter, ManagerBase {
         bytes32 refundAddress,
         bool shouldQueue,
         bytes memory transceiverInstructions
-    ) internal returns (uint64) {
+    ) internal virtual returns (uint64) {
         if (amount == 0) {
             revert ZeroAmount();
         }
